src/storage/mod.rs

```diff
@@ -579,21 +578,17 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
 
         let res = self.read_pool.spawn_handle(
             async move {
-                if let Ok(start_key) = start_key.to_raw() {
-                    let mut key = vec![];
-                    if let Some(end_key) = &end_key {
-                        if let Ok(end_key) = end_key.to_raw() {
-                            key = end_key;
-                        }
-                    }
-                    tls_collect_qps(
-                        ctx.get_region_id(),
-                        ctx.get_peer(),
-                        &start_key,
-                        &key,
-                        reverse_scan,
-                    );
+                let mut key = &vec![];
+                if let Some(end_key) = &end_key {
+                    key = end_key.as_encoded();
```

Comment:
Should follow the similar style with L708.

Reply:
done